src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/SqlConnector.cs

```diff
@@ -19,6 +18,16 @@ namespace Microsoft.Data.SqlClientX
     /// </summary>
     internal class SqlConnector
     {
+        private static int SpoofedServerProcessId = 1;
+
+        internal SqlConnector(SqlConnectionX owningConnection, SqlDataSource dataSource)
+        {
+            OwningConnection = owningConnection;
+            DataSource = dataSource;
+            ServerProcessId = Interlocked.Increment(ref SpoofedServerProcessId);
```

Comment:
Why do we need to increment the Server process Id in client code ?

Reply:
This is just to simulate different server process ids. I added a TODO to clarify and to set it appropriately based on the info reported back during login.